docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -544,19 +544,30 @@ existing Iceberg table from the metastores without deleting the data::
 Migrate table
 """""""""""""
 
-The connector can read from or write to Hive tables that have been migrated to Iceberg.
-The procedure ``system.migrate`` allows the caller to replace
-a Hive table with an Iceberg table, loaded with the source’s data files.
-Table schema, partitioning, properties, and location will be copied from the source table.
-Migrate will fail if any table partition uses an unsupported format::
-
-    CALL iceberg.system.migrate(schema_name => 'testdb', table_name => 'customer_orders')
-
-In addition, you can provide a ``recursive_directory`` argument to migrate a Hive table that contains subdirectories.
-The possible values are ``true``, ``false`` and ``fail``. The default value is ``fail`` that throws an exception
-if nested directory exists::
-
-    CALL iceberg.system.migrate(schema_name => 'testdb', table_name => 'customer_orders', recursive_directory => 'true')
+The connector can read from or write to Hive tables that have been migrated to
+Iceberg.
+
+Use the procedure ``system.migrate`` to move a table from the Hive format to the
+Iceberg format, loaded with the source’s data files. Table schema, partitioning,
```

Comment:
Does that also work for files that are not supported by Iceberg .. for example what if all the files are JSON or CSV or whatever ...

Reply:
No, it doesn't.

Reply:
So only Parquet is supported then right?

Reply:
Updated